### ISSUE TITLE
cloudsearch brings in very old jetty libraries transitively

### ISSUE THREAD

**Issue by davidmankin** [aws/aws-sdk-java-v2#62]
```
$ mvn dependency:tree
+- software.amazon.awssdk:aws-sdk-java:jar:2.0.0-preview-1:compile
|  +- software.amazon.awssdk:cloudsearch:jar:2.0.0-preview-1:compile
|  |  \- com.github.tomakehurst:wiremock:jar:1.55:compile
|  |     +- org.mortbay.jetty:jetty:jar:6.1.26:compile
|  |     |  +- org.mortbay.jetty:jetty-util:jar:6.1.26:compile
|  |     |  \- org.mortbay.jetty:servlet-api:jar:2.5-20081211:compile
```

The org.mortbay classes (especially the servlet-api.jar) are very old versions of the servlet api that (in my case at least) ended up shadowing the modern servlet api, meaning my server wouldn't work after I added the aws-sdk-java dependency. 

I can probably exclude things to make it work, but it seems like cloudsearch shouldn't bring in wiremock at all, let alone letting it bring in an ancient version of jetty.

Browsing the code I don't see anywhere wiremock is used by the cloudsearch jar.

It failed with this stack trace in a dropwizard server:
```
WARN  [2017-07-11 20:49:36,099] org.eclipse.jetty.server.HttpChannel: /swagger.json
! java.lang.NoSuchMethodError: javax.servlet.http.HttpServletResponse.getStatus()I
! at com.codahale.metrics.jetty9.InstrumentedHandler.updateResponses(InstrumentedHandler.java:289)
! at com.codahale.metrics.jetty9.InstrumentedHandler.handle(InstrumentedHandler.java:252)
! at io.dropwizard.jetty.ContextRoutingHandler.handle(ContextRoutingHandler.java:38)
! at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:561)
! at io.dropwizard.jetty.BiDiGzipHandler.handle(BiDiGzipHandler.java:68)
! at org.eclipse.jetty.server.handler.RequestLogHandler.handle(RequestLogHandler.java:56)
! at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:169)
! at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
! at org.eclipse.jetty.server.Server.handle(Server.java:564)
! at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:320)
! at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
! at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
! at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
! at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
! at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672)
! at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590)
! at java.lang.Thread.run(Thread.java:745)
```

**Comment by shorea** [comment]
Oops this should be scoped to test. Good catch, I will fix this.

**Comment by dagnir** [comment]
Looks like this was fixed in #49. Closing.
